### PR TITLE
Print Oryx github repo info when doing a build

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: src/BuildScriptGeneratorCli/Commands/BuildCommand.cs [hunk 1]
```diff
@@ -81,12 +81,16 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
             var buildOpId = logger.StartOperation(appName);
 
             var buildInfo = new DefinitionListFormatter();
-            buildInfo.AddDefinition("Build Operation ID", buildOpId);
             buildInfo.AddDefinition("Oryx Version", $"{Program.GetVersion()}, Commit: {Program.GetCommit()}");
+            buildInfo.AddDefinition("Oryx Repository", "https://github.com/microsoft/oryx");
```
Comment (kichalla) [inline src/BuildScriptGeneratorCli/Commands/BuildCommand.cs]: This is how looks like, FYI:
image

Comment (gpcastro) [reply src/BuildScriptGeneratorCli/Commands/BuildCommand.cs]: I'd use a more free form text, which also explains what Oryx is, e.g. `Build orchestrated by Microsoft Oryx, https://github.com/Microsoft/Oryx/. You can report issues at 'https://github.com/Microsoft/Oryx/Issues'`.
What do you think?

Comment (kichalla) [reply src/BuildScriptGeneratorCli/Commands/BuildCommand.cs]: cc @joshgav

Comment (kichalla) [reply src/BuildScriptGeneratorCli/Commands/BuildCommand.cs]: Updated. Latest looks like below:

image

**Pull request merged**